docs/javascript/v2/advanced-features/hls-stats.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: HLS-Stats
```

Comment:
HLS Stats should be fine. hyphen looks wierd

Reply:
done